_posts/en/resources/2019-09-20-electronic-ultrasonic-sensor-arduino.md

```diff
@@ -0,0 +1,184 @@
+---
+layout:       article
+lang:         en
+parent:       electronic-resources
+breadcrumb:   true
+permalink:    /en/projects/electronic/ultrasonic-sensor-2019
```

Comment:
should be `/en/resources/electronic-resources/ultrasonic-sensor-2019` to be consistent with the sibling pages